diff --git a/coffee.py b/coffee.py
index 8f33893..5817e77 100644
--- a/coffee.py
+++ b/coffee.py
@@ -1,21 +1,22 @@
-class CoffeeCup():
-  def __init__(self, capacity):
-    self.capacity = capacity
-    self.amount = 0
-  
-  def fill(self):
-    self.amount = self.capacity
-  
-  def empty(self):
-    self.amount = 0
-  
-  def drink(self, amount):
-    self.amount -= amount
-    if (self.amount == 0):
-      self.amount = 0
-      
+class CoffeeCup:
+    def __init__(self, capacity: int = 0):
+        self.capacity = capacity
+        self.amount = 0
+
+    def fill(self):
+        self.amount = self.capacity
+
+    def empty(self):
+        self.amount = 0
+
+    def drink(self, amount: int = 0):
+        self.amount -= amount
+        if self.amount == 0:
+            self.amount = 0
+
+
 steves_cup = CoffeeCup(12)  # a fancy latte.
-seans_cup = CoffeeCup(16)    # gas station drip.
+seans_cup = CoffeeCup(16)  # gas station drip.
 brandis_cup = CoffeeCup(2)  # a quick espresso.
 
 print(steves_cup.amount, "ounces in the beginning")
